examples/bert/src/bert_layers.py

```diff
@@ -474,7 +483,18 @@ def forward(
         #        hidden_states[ntokens,hidden] -> hidden_states[ntokens_unpad,hidden]
         hidden_states, indices, cu_seqlens, _ = unpad_input(
             hidden_states, attention_mask_bool)
+
         # Add alibi matrix to extended_attention_mask
+        if self._current_alibi_size < seqlen:
+            # Build/rebuild the alibi tensor if needed
```

Comment:
Should we add a warning every time we do this? Something like:
warnings.warn(f'Increasing max alibi size from {self._current_alibi_size} to {seqlen}')